lib-injection/Dockerfile

```diff
@@ -9,22 +25,52 @@ RUN mkdir -p pkgs
 ADD ./dl_wheels.py .
 # Note that we only get Python >= 3.7. This is to keep the size of the image
 # as small as possible.
+RUN python3 dl_wheels.py \
```

Comment:
this stage can be merged with the previous by using multiple `--python-version`s